pyiso4/lexer.py

```diff
@@ -46,17 +46,17 @@ def __repr__(self) -> str:
 
 
 class Lexer:
-    def __init__(self, inp, stopwords: List[str]):
+    def __init__(self, inp: str, stopwords: List[str]) -> None:
```

Comment:
That should be `"Lexer"` not `None`, then :)

Reply:
`__init__` always returns `None`:
https://docs.python.org/3/reference/datamodel.html#object.__init__

Reply:
Oh, ok, I thought they were returning the Object itself. Good to know :)